docs/docs/quick-start.md

```diff
@@ -6,6 +6,12 @@ This quick start is intended for intermediate to advanced developers. For a gent
 
 ## Use the Gatsby CLI
 
+<iframe class="egghead-video" width=600 height=348 src="https://egghead.io/lessons/gatsby-quick-start-with-gatsby-create-develop-and-build-gatsby-sites-from-the-command-line/embed" />
```

Comment:
Add a `title` to this iframe and it's good to go!

Reply:
Done!